### PR TITLE
WIP: Move Openshift Jenkins CSPI Interop scenario to openshift-ci

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-smatula-jenkins-master-jenkins-ocp4.13-lp-interop-jenkins-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-smatula-jenkins-master-jenkins-ocp4.13-lp-interop-jenkins-interop-aws

**Comment by smatula** [comment]
/pj-rehearse pull-ci-smatula-jenkins-master-jenkins-ocp4.13-lp-interop-images

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-smatula-jenkins-master-jenkins-ocp4.13-lp-interop-jenkins-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-smatula-jenkins-master-jenkins-ocp4.13-lp-interop-jenkins-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-smatula-jenkins-release-4.13-jenkins-hypershift-4.13-lp-interop-aws-rosa-hypershift-jeknis-lp-interop

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-smatula-jenkins-release-4.13-jenkins-hypershift-4.13-lp-interop-aws-rosa-hypershift-jeknis-lp-interop

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-smatula-jenkins-release-4.13-jenkins-hypershift-4.13-lp-interop-aws-rosa-hypershift-jeknis-lp-interop

File: ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-rosa-hypershift-jeknis-lp-interop
+  cron: 0 6 * * 1
```
Comment (ascerra) [inline ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml:53]:
```suggestion
  cron: 0 10 * * 1
```

File: ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-rosa-hypershift-jeknis-lp-interop
```
Comment (ascerra) [inline ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml:52]:
```suggestion
- as: aws-rosa-hypershift-jenkins-lp-rosa-hypershift
```

Comment (ascerra) [reply ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml:52]: You could put the unique identifier `-lp-rosa-hypershift` here like this or you could include this in the filename

File: ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  cron: 0 6 * * 1
+  remote_api: true
```
Comment (ascerra) [inline ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml:54]: We can remove this line

File: ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.14__jenkins-ocp4.14-lp-interop.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: jenkins-interop-aws
+  cron: 0 6 * * 1
+  remote_api: true
```
Comment (ascerra) [inline ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.14__jenkins-ocp4.14-lp-interop.yaml:36]: We can remove this line

File: ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+      OPENSHIFT_VERSION: "4.13"
+      REGION: us-west-2
+      ROSA_LOGIN_ENV: staging
+    test:
+    - ref: jenkins-smoke-tests
+    - ref: cucushift-hypershift-extended-health-check
+    workflow: firewatch-rosa-aws-sts-hypershift
+zz_generated_metadata:
+  branch: release-4.13
+  org: smatula
```
Comment (ascerra) [inline ci-operator/config/smatula/jenkins/smatula-jenkins-release-4.13__jenkins-hypershift-4.13-lp-interop.yaml:79]: Why are we using your personal org here? 

Do we not have a github repo for Jenkins that we can use? If not we might have to break out the hypershift piece into its own PR and just use the rhpit/interop-tests repo so that we can merge the hypershift test asap.

**Comment by smatula** [comment]
Closing this PR... See PR https://github.com/openshift/release/pull/39216

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-smatula-jenkins-release-4.14-jenkins-ocp4.14-lp-interop-jenkins-interop-aws
